core/include/ldml/keyboardprocessor_ldml.ts

```diff
@@ -615,16 +615,18 @@ class Constants {
   }
 
   // ---- marker stuff ----
-  /** sentinel value indicating a marker follows */
-  readonly marker_sentinel = 0xFFFF;
+  /** == UC_SENTINEL, first item of a  */
+  readonly marker_sentinel     = 0xFFFF;
+  /** == CODE_DEADKEY */
+  readonly marker_code_deadkey = 0x0008;
```

Comment:
```suggestion
  readonly marker_code = 0x0008;
```

Or perhaps code_marker ... up for discussion!

Reply:
for future extension I guess code_marker, could then match code_beep, code_return  etc